### DISCUSSION TITLE
Finally got an error for my previous question

### DISCUSSION THREAD

**Discussion by Manniac07** [vectordotdev/vector#19302] [Q&A]
this is in regards to my [previous question](https://github.com/vectordotdev/vector/discussions/19261)
I finally got an error I am hoping  someone might be able to respond to this now that I have an error

`2023-12-04T19:47:31.247430Z  WARN sink{component_kind="sink" component_id=mysink_id_writer component_type=prometheus_remote_write component_name=mysink_id_writer}:request{request_id=51614}:http: vector::internal_events::http_client: HTTP error. error=connection closed before message completed error_type="request_failed" stage="processing" internal_log_rate_limit=true
2023-12-04T19:47:31.254703Z  WARN sink{component_kind="sink" component_id=mysink_id_writer component_type=prometheus_remote_write component_name=mysink_id_writer}:request{request_id=51614}: vector::sinks::util::retries: Retrying after error. error=Failed to make HTTP(S) request: connection closed before message completed internal_log_rate_limit=true`

**Comment by jszwedko** [comment]
Hi @Manniac07 !

This can happen if the connection is terminated unexpectedly. Vector will automatically retry this error, though, so you can ignore it. https://github.com/vectordotdev/vector/issues/15196 is tracking dropping the level of this log to `debug` since tends to be a red herring.

**Reply by Manniac07** [reply]
@jszwedko So then I am still at square one trying to figure out why remote write slowly stops sending metrics to Victoria metrics.
